README.md

```diff
@@ -261,24 +267,54 @@ and vice-versa. We have created this [codesandbox](https://codesandbox.io/s/213m
 
 Let's add some children to the AppBar to make it more realistic.
 
-Let import `Button`, `Heading`, and `Notification` icon.
+Let import `Button`, `Header`, `Heading`, and `Notification` icon.
 
 ```diff
 - import { Box, Grommet } from 'grommet';
-+ import { Box, Button, Heading, Grommet } from 'grommet';
++ import { Box, Button, Header, Heading, Grommet } from 'grommet';
 + import { Notification } from 'grommet-icons';
 ```
 
+Now we can swap out the Box in the AppBar for the Header. Header is a specialized version of the Box component that defines some of the props commonly used for a header component automatically.
```

Comment:
Can we just start with `Header` ? like below we say `using layer` here can we just say `using header` and explain that `header` is a `box`? seems a lot to add this `box` just to turn around to replace it 
## Using Layer

Reply:
Done in latest commit